content/200-concepts/100-components/04-introspection.mdx

```diff
@@ -54,21 +54,14 @@ Here's a high-level overview of the steps that `prisma db pull` performs interna
 
 ## Introspection workflow
 
-<Admonition type="warning">
-
-**MongoDB not supported** <br />
-For now, using MongoDB introspection is meant to be done only once for initial data model. Running it repeatedly will lead to loss of data in the schema file, such as relations or comments.
-
-</Admonition>
-
 The typical workflow for projects that are not using Prisma Migrate, but instead use plain SQL or another migration tool looks as follows:
 
 1. Change the database schema (e.g. using plain SQL)
 1. Run `prisma db pull` to update the Prisma schema
 1. Run `prisma generate` to update Prisma Client
 1. Use the updated Prisma Client in your application
 
-Note that as you evolve the application, this process can be repeated for an indefinite number of times.
+Note that as you evolve the application, [this process can be repeated for an indefinite number of times](#introspection-with-an-existing-schema).
```

Comment:
detail removed here, but linked to section that includes details about this instead